static/js/stream_create.js

```diff
@@ -364,7 +364,8 @@ exports.create_handlers_for_users = function (container) {
             // implementation is merely sluggish.
             user_labels.each(function () {
                 const elem = $(this);
-                const user_id = elem.attr('data-user-id');
+                let user_id = elem.attr('data-user-id');
+                user_id = parseInt(user_id, 10);
```

Comment:
Prefer `const user_id = parseInt(elem.attr('data-user-id'), 10)`. The type-shifting mutation is unnecessary, and TypeScript won’t like it when it becomes TypeScript.

Reply:
fixed